src/Libraries/CoreNodeModels/Selection.cs

```diff
@@ -102,8 +102,12 @@ public virtual string SelectionSuggestion
 
             this.selectionType = selectionType;
             this.selectionObjectType = selectionObjectType;
-
-            OutPortData.Add(new PortData("Elements", Resources.SelectionPortDataResultToolTip));
+            if (selectionMessage.Equals("Select Model Element"))
+            {
+                OutPortData.Add(new PortData("Element", Resources.SelectionPortDataResultToolTip));
+            }
+            else 
+                OutPortData.Add(new PortData("Elements", Resources.SelectionPortDataResultToolTip));
```

Comment:
1. in if else statement :one is "Element",the other  is "Elements". It is to create different outports.

2.selectionMessage is from the SelectionBase, which is a parameter in SelectionBase method.